### PR TITLE
Separate project for RecurringTasks

### PR DESCRIPTION
**Pull request by isaac73**
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java [hunk 1]
```diff
@@ -29,21 +30,21 @@
...
-    public void run() {
-        this.buildCache();
-        this.scheduleCacheUpdate();
+    public void Run() {
+        this.BuildDevicePropertiesCache();
+        this.ScheduleDevicePropertiesCacheUpdate();
     }
 
-    private void buildCache() {
+    private void BuildDevicePropertiesCache() {
```
Comment (ppathan) [inline app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java]: I looked at other files in this project and we use camelCase for the method names for most part. Right now at few places it is mix of both. Can you do a scan and fix it, i think it should be camelCase. Unless you find anything different.

File: app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java [hunk 2]
```diff
@@ -29,21 +30,21 @@
...
         this.cache = cache;
-        CompletableFuture.runAsync(() -> this.run());
+        CompletableFuture.runAsync(() -> this.Run());
     }
 
     @Override
-    public void run() {
-        this.buildCache();
-        this.scheduleCacheUpdate();
+    public void Run() {
```
Comment (molly-moen) [inline app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java]: camel case?

File: app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java (hunk 1 above)
Comment (isaac73) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java]: Just came to know method names are camelCase in Java and PascalCase in dotnet. 
Done

File: app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java (hunk 2 above)
Comment (isaac73) [reply app/com/microsoft/azure/iotsolutions/iothubmanager/RecurringTasksAgent/RecurringTasks.java]: done

**Pull request merged**